front/app/components/SeatInfo/messages.ts

```diff
@@ -17,22 +17,32 @@ export default defineMessages({
     id: 'app.components.SeatInfo.additionalSeats',
     defaultMessage: 'Additional seats',
   },
-  adminMessage: {
-    id: 'app.components.SeatInfo.adminMessage2',
+  adminInfoTextWithBilling: {
+    id: 'app.components.SeatInfo.adminInfoTextWithBilling',
     defaultMessage:
       "Your plan has {adminSeatsIncluded}. Once you've used all the seats, you agree on us billing the additional seats according to your license.",
   },
-  adminSeatsIncludedSubText: {
-    id: 'app.components.SeatInfo.adminSeatsIncludedSubText',
+  adminInfoTextWithoutBilling: {
+    id: 'app.components.SeatTrackerInfo.adminInfoTextWithoutBilling',
+    defaultMessage:
+      "Your plan has {adminSeatsIncluded}. Once you’ve used all the seats, extra seats will be added under 'Additional seats'.",
+  },
+  adminSeatsIncludedText: {
+    id: 'app.components.SeatInfo.adminSeatsIncludedText',
     defaultMessage: '{adminSeats} admin seats included',
   },
-  collaboratorMessage: {
-    id: 'app.components.SeatInfo.collaboratorMessage2',
+  collaboratorInfoTextWithBilling: {
+    id: 'app.components.SeatInfo.collaboratorInfoTextWithBilling',
+    defaultMessage:
+      "Your plan has {collaboratorSeatsIncluded}, eligible for folder managers and project managers. Once you've used all the seats, you agree on us billing the additional seats according to your license.",
+  },
+  collaboratorInfoTextWithoutBilling: {
+    id: 'app.components.SeatTrackerInfo.collaboratorInfoTextWithoutBilling',
     defaultMessage:
-      "Your plan has {adminSeatsIncluded}, eligible for folder managers and project managers. Once you've used all the seats, you agree on us billing the additional seats according to your license.",
+      "Your plan has {collaboratorSeatsIncluded}, eligible for folder managers and project managers. Once you’ve used all the seats, extra seats will be added under 'Additional seats'.",
```

Comment:
```suggestion
      "Your plan has {collaboratorSeatsIncluded}, eligible for folder managers and project managers. Once you've used all the seats, extra seats will be added under 'Additional seats'.",
```

This is an italic quote again. The ones around "Additional seats" are straight quotes tho. Do you know how you add them? Or was it copy & paste?